api-gateway/src/main/resources/application.yml

```diff
@@ -0,0 +1,43 @@
+server:
+  port: 5300
+gateway:
+  backend:
+    url: http://backend:5000
+management:
+  endpoints:
+    web:
+      exposure:
+        include: health,info,prometheus,gateway
+spring:
+  profiles:
+    default: dev
+  cloud:
+    gateway:
+      routes:
+        - id: api_route
+          uri: http://backend:5000
+          predicates:
+            - Path=/api/**
+          filters:
+            - StripPrefix=1
+            # If SESSION cookie is passed to downstream, it is then removed, because downstream discards it
```

Comment:
It was needed for credentials-based authentication, need to check if it can be removed for OAuth

Reply:
Spring still sets `SESSION` cookie, so I'll leave it here at least till #417